solver_greedy_max.py

```diff
@@ -43,7 +68,6 @@
    
     sites_by_obs = {obs_id: 'gs' if sum(ttab_gngs['weight_gs'][idx]) > 0 else 'gn' for idx, obs_id in enumerate(obs_ids)}
```

Comment:
I'd still like to see this changed to `Site.GS` and `Site.GN` instead of the strings.